### DISCUSSION TITLE
Migration CCU3 zu ProxmoxVE mit HmIP-DRAP und HmIP-RFUSB

### DISCUSSION THREAD

**Discussion by bandimuc** [OpenCCU/OpenCCU#2695] [🇩🇪 Fragen&Antworten] [answered]
Hallo zusammen,

ich hätte da mal eine Frage an die Experten. Ich bin gerade dabei meine CCU3 gegen einen Proxmox NUC auszutauschen. Bei dem ersten System, wo nur Funk im Einsatz ist, lief alles problemlos. Backup gezogen, in das neue System eingespielt, läuft :)

Nun möchte ich das Gleiche bei meinem HmIP Wire System machen. Den RFUSB habe ich der VM über USB passthrough zugewiesen. Nach einspielen des Backups laufen alle Funkgeräte problemlos, nun der DRAP wird vom System nicht erkannt and blinkt hektisch blau ?!?

Habe ich hier etwas übersehen ? (FW wurde bei der VM bereits deaktiviert)

Sachdienliche Hinweise sehr willkommen :)

Danke Euch im Voraus
VG Bernhard

**Comment by bandimuc** [comment]
Hallo zusammen, ein kurzer Zwischenstand. Nachdem ich das komplette Wired Bussystem rebooted habe, hat nun die CCUprox den DRAP gefunden und die LED leuchtet blau :)

Ich warte mal 48h ob es so bleibt, wenn ja, schließe ich die Frage...

**Comment by bandimuc** [comment]
Umstellung auf CCU Proxmox mit RFUSB Stick hat mit Restore super geklappt, jedoch muss im Vergleich zu einem reinen Funksystem das Wired System nach dem Umstellung einmal komplett rebooted werden, damit die CCU den DRAP erkennt. System läuft nun 48h fehlerfrei und auch nach etlichen reboots der CCU wurde der DRAP immer wieder ohne Probleme erkannt. Case closed :)

**Comment by hd-schardt** [comment]
Hallo,
da ich den DRAP ebenfalls nicht an Raspberrymatic angelernt bekomme, möchte ich mich hier anschließen.
Raspberrymatic läuft unter Proxmox als VM
gehe ich in den Anlernmodus , so erhalte ich folgende Meldung

![78A62C54-2DEB-4557-8531-0484A1DEC3DD_1_105_c](https://github.com/jens-maus/RaspberryMatic/assets/61353303/6a417989-5783-4d48-887f-071d0e29b2fa)

Ein Funkmodul oder Stick ist nicht vorhanden, weil die gesamte Anlage über Wired-Komponenten betrieben soll. Stellt sich mir jetzt die Frage, wie bekomme ich den Wired-Accesspoint in Raspberrymatic integriert. Zwar habe ich schon im Internet nach Lösungen gesucht, aber bisher erfolglos.

In der HomematicIP App konnte ich den DRAP ohne Probleme einbinden, deshalb gehe ich davon aus, dass es am Netzwerk nicht liegt.

Grüße Heinz-Dieter

**Reply by jens-maus** [reply, marked as answer]
> da ich den DRAP ebenfalls nicht an Raspberrymatic angelernt bekomme, möchte ich mich hier anschließen.
> [...]
> Ein Funkmodul oder Stick ist nicht vorhanden, weil die gesamte Anlage über Wired-Komponenten betrieben soll.

Das ist ein leider häufig auftretender Trugschluss. Bei dem RPI-RF-MOD, HmIP-RFUSB und den anderen homematicIP "Funkmodulen" handelt es sich um mehr als nur reine Funkmodule! Dies sind Coprozessoren die für gewisse Verschlüsselungs- und Routingfunktionen des homematicIP Protokolles zwingend benötigt werden. Ergo: Egal ob du wirklich Funkbasierte homematicIP Geräte einsetzen willst oder nicht, ein solches RPI-RF-MOD oder HmIP-RFUSB wird **immer** benötigt!

> In der HomematicIP App konnte ich den DRAP ohne Probleme einbinden, deshalb gehe ich davon aus, dass es am Netzwerk nicht liegt.

Das ging nur weil der Cloud-Gateway (HmIP-HAP) über solch ein "Funkmodul" bzw. Coprozessor intern verfügt wie das das RPI-RF-MOD oder HmIP-RFUSB bietet.

**Comment by hd-schardt** [comment]
Vielen Dank für die aufschlussreiche Antwort. Da hätte ich lange suchen können, da ich mangels eingehenden Wissens von falschen Voraussetzungen ausging. Das hat mir sehr weitergeholfen.

Grüße
Heinz-Dieter

**Reply by jens-maus** [reply]
All dies kann man auch einfach aus der offiziellen Dokumentation herauslesen wenn man diese aufmerksam studiert. So z.b. ist im Abschnitt "Voraussetzungen" recht detailliert in einer Tabelle die möglichen Kombinationen und Abhängigkeiten der "Funkmodule" und Gateways ablesen:

https://github.com/jens-maus/RaspberryMatic/wiki/Einleitung#voraussetzungen

**Comment by ivo7B4** [comment]
Bei mir hatte dies auch mit den Funkgeräten super geklappt, aber nicht mit den wired Geräten. 
Ich hatte jedoch nicht rebooted: "jedoch muss im Vergleich zu einem reinen Funksystem das Wired System nach dem Umstellung einmal komplett rebooted werden, damit die CCU den DRAP erkennt."
Ist dies erforderlich? Wenn ja, wäre super, wenn dies in der Beschreibung noch ergänzt würde, vielen Dank! (und entschuldige mich, falls dies dort bereits vorhanden und ich nur übersehen habe).
